src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -1,16 +1,57 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+// Handles fetching and saving user data.
 @WebServlet("/about")
 public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  // Responds with the "about me" section for a particular user.
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Responds with a hard-coded message for testing purposes.
-    response.getOutputStream().println("hello world");
+
+    response.setContentType("text/html");
+
+    String user = request.getParameter("user");
+
+    if (user == null || user.equals("")) {
+      // Request is invalid, return empty response
+      return;
+    }
+
+    String aboutMe = "This is " + user + "'s about me.";
```

Comment:
Use `String.format("This is %s's about me.", user);`. This makes it slightly easier to read the sentence, but more importantly, if we need to localize the entire string (like internationalization which we don't cover in CodeU) then it will need to be in a formatted way instead of broken up strings concatenated using +.